### PR TITLE
added webpack and babel configuration

### PR DESCRIPTION
**Pull request by XSpiritBalanceX**
webpack and babel configuration, react app deployment

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    publicPath: "/dist",
+    assetModuleFilename: "assets/[hash][ext][query]",
+  },
+  devServer: {
+    hot: true,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
```
Comment (AlehPanasiuk) [inline webpack.config.js]:
```suggestion
        test /\.(js|jsx)$/,
```
We can write some helpers and other files)

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: "css-loader",
+          },
+          {
+            loader: "sass-loader",
+          },
+        ],
+      },
```
Comment (AlehPanasiuk) [inline webpack.config.js]:
```suggestion
      {
        test: /\.(sa|sc|c)ss$/,
        use: [
          MiniCssExtractPlugin.loader,
          {
            loader: "css-loader",
          },
          {
            loader: "sass-loader",
          },
        ],
      },
```

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        },
+      },
+      {
+        test: /\.css$/,
+        use: [MiniCssExtractPlugin.loader, "css-loader"],
+      },
+      {
+        test: /\.scss$/,
+        use: [
+          MiniCssExtractPlugin.loader,
```
Comment (AlehPanasiuk) [inline webpack.config.js:59]: Do you have checked if this plugin will load the CSS or scss files?

Comment (XSpiritBalanceX) [reply webpack.config.js:59]: yes. In my previous project

**Review by OlegPanasyuk — APPROVED** [review]

**Pull request merged**